docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -28,19 +28,37 @@ Combobox is composed of three main parts:
 #### Trigger
 
 The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
-
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+initiate option selection. When implementing Combobox, you may rely on the
+default trigger or, you may pass in a custom activator.
+
+The default trigger is a
+[Chip](../?path=/story/components-selections-chip-web--subtle) containing a
+label that reads "Select" and an `add` Icon suffix. The label can be customized
+by adding the optional `label` prop to better reflect the type of data being
+selected. Examples of using a default trigger can be found in the
+[single-select](../?path=/code/components-selections-combobox-web--single-select)
 and
-[TriggerChip](../?path=/story/components-selections-combobox-web--trigger-chip).
-You must have one trigger component.
+[multi-select](../?path=/code/components-selections-combobox-web--multi-select)
+stories.
+
+Notice that omitting a trigger subcomponent will result in the default trigger
+being used.
+
+The custom activator should be used when the Combobox trigger is a
+[Button](../?path=/docs/components-actions-button-docs--page) or an alternate
+variation of Chip. Button or Chip should be a child of `<Combobox.Activator />`
+and will be rendered in place of the default trigger. Examples of using a custom
+activator can be found in the
+[custom activator](../?path=/code/components-selections-combobox-web--custom-activator)
+story.
 
 #### Content
 
-This component is responsible for displaying the list of selectable options,
-handling user interactions, and managing search functionality.
+The content of the Combobox is responsible for displaying the list of selectable
+options, handling user interactions, and managing search functionality.
+
+`<Combobox.Option />` is a wrapper component that should be used to wrap each
```

Comment:
I'd change the wording here since at this point in time, `Combobox.Option` does not support children (lol what a sentence)

all of its functionality comes solely from its props, and we don't need people to wrap anything with the `Option`

one day when we support headers or custom format that might change, but let's wait til we cross that bridge!

Reply:
Ok I kept it very surface level for now, but changed this sentence here e6db88b642dc292eb549e6a52b3de510d8a599ca